libclamav/pe.c

```diff
@@ -2468,28 +2468,30 @@ static cl_error_t hash_imptbl(cli_ctx *ctx, unsigned char **digest, uint32_t *im
         }
     }
 
-    image = (struct pe_image_import_descriptor *)impdes;
-    while (left > sizeof(struct pe_image_import_descriptor) && image->Name != 0 && nimps < PE_MAXIMPORTS) {
+    memcpy(&image, impdes, sizeof(struct pe_image_import_descriptor));
```

Comment:
`impdes` is declared `const char *` above, but received from `fmap_need_off()`, which returns `void *`.  Recommend aligning these types (`struct pe_image_import_descriptor *`)

Reply:
Aligning the types for the `image` and `impdes` pointers sounds good to me.